test/integration/test_cdc.py

```diff
@@ -15,7 +15,8 @@
 from superduperdb.db.base.cdc import DatabaseListener
 from superduperdb.db.mongodb.query import Collection
 from superduperdb.vector_search.base import VectorCollectionConfig
-from superduperdb.vector_search.lancedb_client import LanceVectorIndex
+#from superduperdb.vector_search.lancedb_client import LanceVectorIndex
```

Comment:
comments?

Reply:
I removed the `lancedb_client` module in this PR. I left it as a comment for now as a hint about what needs to be changed.